examples_new/input_default.json

```diff
@@ -4,7 +4,7 @@
   "description": "Example FLORIS Input file",
   "logging": {
     "console": {
-      "enable": true,
+      "enable": false,
```

Comment:
My thought was to have console logging turned on for new users, so they can see any messages/info. What do you think?